src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -2373,6 +2471,13 @@ public static function get_tests() {
 			);
 		}
 
+		if ( 'production' === wp_get_environment_type() ) {
+			$tests['direct']['persistent_object_cache'] = array(
```

Comment:
Should this perhaps be an `async` test, given the potential DB load from `should_suggest_persistent_object_cache()`?

Reply:
@manuelRod: I wrote this 🙃

Reply:
ohh @tillkruss sorry for that, i mixed who wrote what :)

Reply:
@tillkruss should we do this kind of big change, first on the plugin level, and then move them here?

Reply:
Yeah, maybe we do the badge color and async change on `wordpress/performance` first? Do you want to open a PR, or shall I?

Reply:
@Clorith as @tillkruss comments in the next comment block, this query is super fast, I don't think we need to move to async either.

Reply:
Agreed, while I'm not opposed to making the check async, I don't think it's worth it either. Looking at existing Site Health checks, it appears like the policy for async tests is mostly around any external HTTP requests, which makes sense. Cheap database lookups like here however don't justify the additional effort IMO.